Sources/ArgumentParser/Usage/MessageInfo.swift

```diff
@@ -9,9 +9,14 @@
 //
 //===----------------------------------------------------------------------===//
 
-#if swift(>=5.11)
+#if swift(>=6.0)
```

Comment:
See previous comment that Swift 6 language mode needs to be enabled for these guards to work (which it is not in Package.swift).

Reply:
Good catch, thanks @MaxDesiatov!